src/main.cpp

```diff
@@ -32,7 +32,9 @@ int main(int argc, char** argv)
 	
 	// Create the game and run the main loop
 	FGame game(engine);
+	game.loadCharacterType(FCharacterType::DEBUG,
+	"images/business.png"); // or all the available filenames should be saved as in string (without loading images) and just use FCharacterType?
```

Comment:
I mean there might be n different textures saved in the directory images. When we run the game, it could check the files in the directory and save the existing names in string with predefined FCharacterType. So in this case, we will just use the FCharacterType and it will load the right sprite anyway.